integration_tests/CMakeLists.txt

```diff
@@ -780,3 +780,4 @@ COMPILE(NAME import_order_01     LABELS cpython llvm c) # any
 
 # LPython emulation mode
 RUN(NAME lpython_emulation_01    LABELS cpython NOMOD)
+RUN(NAME lpython_emulation_02    LABELS cpython)
```

Comment:
Add `llvm` test as well. And rename the test, it should not be called emulation --- the emulation test is only for CPython only tests, which we have just one, to show how to hook your own implementations in CPython.

I think this is a "struct" test.

Reply:
Unless this test doesn't run in LPython yet, in which case keep this test cpython only and let's also add a second test that runs in LPython and still breaks in CPython 3.11.

Reply:
>still breaks in CPython 3.11.

If we add a test that breaks, the CI might fail and we might not be able to merge.